build.sbt

```diff
@@ -30,7 +30,7 @@ val installAll =
   s"""apk update && apk add bash curl &&
      |apk add --update ruby ruby-bundler ruby-dev &&
      |rm /var/cache/apk/* &&
-     |gem install brakeman""".stripMargin.replaceAll(System.lineSeparator(), " ")
+     |gem install brakeman --no-rdoc --no-ri""".stripMargin.replaceAll(System.lineSeparator(), " ")
```

Comment:
In the version of gem at least that is being used in your docker instances,
those flags should not be deprecated. In fact, I can't even built the
instance without specifying them, which is why I added them.

On Mon, Jan 11, 2016 at 9:15 AM, Joao Machado <notifications@github.com>
wrote:

> In build.sbt
> <https://github.com/codacy/codacy-brakeman/pull/3#discussion_r49348308>:
>
> > @@ -30,7 +30,7 @@ val installAll =
> >    s"""apk update && apk add bash curl &&
> >       |apk add --update ruby ruby-bundler ruby-dev &&
> >       |rm /var/cache/apk/* &&
> > -     |gem install brakeman""".stripMargin.replaceAll(System.lineSeparator(), " ")
> > +     |gem install brakeman --no-rdoc --no-ri""".stripMargin.replaceAll(System.lineSeparator(), " ")
>
> Why do we need those flags? They seem to be deprecated
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codacy/codacy-brakeman/pull/3/files#r49348308>.
>



-- 
Justice London
jlondon@syrussystems.com